### PR TITLE
fix(blog): fix images in post

### PR DESCRIPTION
**Pull request by MichaelSolati**

### CODE REVIEW

File: src/site/content/en/blog/sign-up-form-best-practices/index.md [hunk 1]
```diff
@@ -275,7 +268,7 @@ Make sure to [salt and hash](https://cheatsheetseries.owasp.org/cheatsheets/Pass
 
 ## Don't force password updates {: #no-forced-password-updates}
 
-[Don't force users to update their passwords arbitrarily.](https://pages.nist.gov/800-63-3/sp800-63b.html#-5112-memorized-secret-verifiers:~:text=Verifiers%20SHOULD%20NOT%20require%20memorized%20secrets%20to%20be%20changed%20arbitrarily%20(e.g.%2C%20periodically).)
+[Don't force users to update their passwords arbitrarily.](<https://pages.nist.gov/800-63-3/sp800-63b.html#-5112-memorized-secret-verifiers:~:text=Verifiers%20SHOULD%20NOT%20require%20memorized%20secrets%20to%20be%20changed%20arbitrarily%20(e.g.%2C%20periodically).>)
```
Comment (robdodson) [inline src/site/content/en/blog/sign-up-form-best-practices/index.md:271]: are the angle brackets correct here?

**Review by robdodson — APPROVED** [review]

**Comment by robdodson** [comment]
LGTM with one comment

File: src/site/content/en/blog/sign-up-form-best-practices/index.md (hunk 1 above)
Comment (samdutton) [reply src/site/content/en/blog/sign-up-form-best-practices/index.md:271]: No. Not sure where this came from — looks OK in the index.md file as it is at the moment.

Comment (MichaelSolati) [reply src/site/content/en/blog/sign-up-form-best-practices/index.md:271]: Sorry, my editor autoformats on save. I'll try to undo this.

Comment (robdodson) [inline src/site/content/en/blog/sign-up-form-best-practices/index.md:271]:
```suggestion
[Don't force users to update their passwords arbitrarily.](https://pages.nist.gov/800-63-3/sp800-63b.html#-5112-memorized-secret-verifiers:~:text=Verifiers%20SHOULD%20NOT%20require%20memorized%20secrets%20to%20be%20changed%20arbitrarily%20(e.g.%2C%20periodically).)
```

**Review by robdodson — APPROVED** [review]

**Pull request merged**